src/ResultWriter/faultoutput.f90

```diff
@@ -642,7 +642,7 @@ SUBROUTINE calc_FaultOutput( DynRup_output, DISC, EQN, MESH, MaterialVal, BND, t
                   ENDDO
                 ENDDO
                 DO iDegFr = 1,nDegFr2d
-                   projected_RT(iDegFr) =  projected_RT(iDegFr)/DISC%Galerkin%MassMatrix_Tri(iDegFr,iDegFr,DISC%Galerkin%nPoly)
+                   !projected_RT(iDegFr) =  projected_RT(iDegFr)/DISC%Galerkin%MassMatrix_Tri(iDegFr,iDegFr,DISC%Galerkin%nPoly)
```

Comment:
are you sure about that?
e.g. C. Pelties dissertation, equation 2.7

Reply:
Please don't review yet. I commented that because I removed MassMatrix_Tri and I need to replace that by something else. (One of the TODOs.)

Reply:
ok sorry

Reply:
No need to apologize. I appreciate the effort :)